recipes/rapidcheck/all/conandata.yml

```diff
@@ -5,3 +5,6 @@ sources:
   "20210107":
     url: "https://github.com/emil-e/rapidcheck/archive/37bcab858dfc398634d409c973363088703ba96e.zip"
     sha256: "35433d0417fb945abe2b56b720b03d6d92da1188e760c50662c243dac42a6717"
+  "20210702":
```

Comment:
Please add a `cci.` prefix to all packages.
I think the approach of https://github.com/conan-io/conan-center-index/pull/6951 can be re-used here.

Reply:
Better if we create a recipe-revision for _old_ versions with a message to inform consumer about the new reference they should start using: https://github.com/dmn-star/conan-center-index/pull/1

Reply:
Wouldn't it be possible to simply delete the old revision in that case?

Reply:
The _issue_ is that anyone using `rapidcheck.20210107` will keep using it _forever_. The center will return the latest recipe-revision available for that reference. A new recipe-revision for a different reference (even if it is the same library name) doesn't invalidate other versions.

The path here is to create a new recipe revision (that will become the latest one) with the warning/error message, people trying to update `rapidcheck.20210107` will get the error and can decide to move to the `.cci` version and new recipe revisions.

Reply:
> Wouldn't it be possible to simply delete the old revision in that case?

Sorry, I read _version_ not revision. No, this is not possible. We don't remove anything from ConanCenter.

Reply:
ok